nixos/modules/virtualisation/containers.nix

```diff
@@ -134,6 +134,9 @@ let
         --setenv HOST_ADDRESS6="$HOST_ADDRESS6" \
         --setenv LOCAL_ADDRESS6="$LOCAL_ADDRESS6" \
         --setenv PATH="$PATH" \
+        ${if cfg . "additionalCapabilities" or null != null then
+          ''--capability="${concatStringsSep " " cfg . "additionalCapabilities"}"'' else ""
```

Comment:
Commit b9df84cd4f6b70bdfa395a91dbf3d712adc7e18d gets rid of the need for the 'or null`. You'll need to add `additionalCapabilities` to `dummyConfig` though.